docs/sources/reference/components/mimir/mimir.rules.kubernetes.md

```diff
@@ -234,8 +237,8 @@ mimir.rules.kubernetes "local" {
 }
 ```
 
-This example creates a `mimir.rules.kubernetes` component that loads discovered
-rules to Grafana Cloud.
+This example creates a `mimir.rules.kubernetes` component that loads discovered rules to Grafana Cloud. 
+It will also add a `"label1"` label to each rule. If that label already exists, its value will be overridden with `"value1"`.
```

Comment:
```suggestion
It also adds a `"label1"` label to each rule. If that label already exists, it is overwritten with `"value1"`.
```

It's still passive... but good enough since we are describing what happens in the example.